satellite/console/consoleweb/server.go

```diff
@@ -247,107 +248,107 @@ func (s *Server) createRegistrationTokenHandler(w http.ResponseWriter, req *http
 }
 
 // accountActivationHandler is web app http handler function
-func (s *Server) accountActivationHandler(w http.ResponseWriter, req *http.Request) {
-	ctx := req.Context()
+func (server *Server) accountActivationHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
 	defer mon.Task()(&ctx)(nil)
-	activationToken := req.URL.Query().Get("token")
+	activationToken := r.URL.Query().Get("token")
 
-	err := s.service.ActivateAccount(ctx, activationToken)
+	err := server.service.ActivateAccount(ctx, activationToken)
 	if err != nil {
-		s.log.Error("activation: failed to activate account",
+		server.log.Error("activation: failed to activate account",
 			zap.String("token", activationToken),
 			zap.Error(err))
 
-		s.serveError(w, req)
+		server.serveError(w, r)
 		return
 	}
 
-	http.ServeFile(w, req, filepath.Join(s.config.StaticDir, "static", "activation", "success.html"))
+	http.ServeFile(w, r, filepath.Join(server.config.StaticDir, "static", "activation", "success.html"))
 }
 
-func (s *Server) passwordRecoveryHandler(w http.ResponseWriter, req *http.Request) {
-	ctx := req.Context()
+func (server *Server) passwordRecoveryHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
 	defer mon.Task()(&ctx)(nil)
-	recoveryToken := req.URL.Query().Get("token")
+	recoveryToken := r.URL.Query().Get("token")
 	if len(recoveryToken) == 0 {
-		s.serveError(w, req)
+		server.serveError(w, r)
 		return
 	}
 
-	switch req.Method {
+	switch r.Method {
 	case http.MethodPost:
-		err := req.ParseForm()
+		err := r.ParseForm()
 		if err != nil {
-			s.serveError(w, req)
+			server.serveError(w, r)
 			return
 		}
 
-		password := req.FormValue("password")
-		passwordRepeat := req.FormValue("passwordRepeat")
+		password := r.FormValue("password")
+		passwordRepeat := r.FormValue("passwordRepeat")
 		if strings.Compare(password, passwordRepeat) != 0 {
-			s.serveError(w, req)
+			server.serveError(w, r)
 			return
 		}
 
-		err = s.service.ResetPassword(ctx, recoveryToken, password)
+		err = server.service.ResetPassword(ctx, recoveryToken, password)
 		if err != nil {
-			s.serveError(w, req)
+			server.serveError(w, r)
 			return
 		}
 
-		http.ServeFile(w, req, filepath.Join(s.config.StaticDir, "static", "resetPassword", "success.html"))
+		http.ServeFile(w, r, filepath.Join(server.config.StaticDir, "static", "resetPassword", "success.html"))
 	case http.MethodGet:
-		t, err := template.ParseFiles(filepath.Join(s.config.StaticDir, "static", "resetPassword", "resetPassword.html"))
+		t, err := template.ParseFiles(filepath.Join(server.config.StaticDir, "static", "resetPassword", "resetPassword.html"))
```

Comment:
This would also lead to an parsing of the template, everytime someone is requesting that page. Cant we do that on startup instead?